notebooks/Model_Training_with_BERT.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>Take a look at Analyze_Model_Outputs.ipynb for an example of drilling down to problematic documents based on document-level precision and recall numbers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
I'll try to cover this as a followup